.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tock"]
 	path = tock
-	url = https://github.com/helena-project/tock.git
+	url = https://github.com/tock/tock.git
```

Comment:
I think at some point you should have tock-os.com or tock.com as a standard url, rather than github. Just a thought.